src/app/components/Map/Map.tsx

```diff
@@ -77,41 +89,50 @@ export default function MapBox(): JSX.Element {
       },
     }
 
-    const pointData = {
+    const pointData: GeoJSON.FeatureCollection<GeoJSON.Geometry> = {
       type: 'FeatureCollection',
       features: [
         {
           type: 'Feature',
           properties: {},
           geometry: {
             type: 'Point',
-            coordinates: location1,
+            coordinates: location1 as GeoJSON.Position,
           },
         },
       ],
     }
-    map.current
-      ? map.current.fitBounds([location1, location2], {
-          padding: { top: 100, bottom: 100, left: 100, right: 100 },
-        })
-      : null
 
-    const endData = {
+    const endData: GeoJSON.FeatureCollection<GeoJSON.Geometry> = {
       type: 'FeatureCollection',
       features: [
         {
           type: 'Feature',
           properties: {},
           geometry: {
             type: 'Point',
-            coordinates: location2,
+            coordinates: location2 as GeoJSON.Position,
           },
         },
       ],
     }
 
-    if (map.current?.getSource('route')) {
-      map.current.getSource('route').setData(geojson)
+    map.current && location1 && location2
```

Comment:
I think an if statement would be nicer and more fitting given the current code style